Description: Fix port already in use test failure
 Between the IPv4 and IPv6 tests we need to bump the port number
 to prevent the test suite failing and consequently hanging the 
 build process
Author: Daniel Lintott <daniel@serverb.co.uk>
Last-Update: 2014-05-30
--- a/t/01live.t
+++ b/t/01live.t
@@ -32,6 +32,7 @@
 my @classes = (qw(HTTP::Server::Simple SlowServer));
 for my $class (@classes) {
     run_server_tests($class, AF_INET);
+    $PORT++;
     run_server_tests($class, AF_INET6);
     $PORT++; # don't reuse the port incase your bogus os doesn't release in time
 }
